pkg/cmd/server/kubernetes/node_config_test.go

```diff
@@ -99,11 +99,18 @@ func TestKubeletDefaults(t *testing.T) {
 			OutOfDiskTransitionFrequency:   unversioned.Duration{Duration: 5 * time.Minute},
 			HairpinMode:                    "promiscuous-bridge",
 			BabysitDaemons:                 false,
+			SeccompProfileRoot:             "/var/lib/kubelet/seccomp",
+			CloudProvider:                  "auto-detect",
+			RuntimeRequestTimeout:          unversioned.Duration{Duration: 2 * time.Minute},
+			ContentType:                    "application/vnd.kubernetes.protobuf",
```

Comment:
Are we overwriting this entire part of the node config or does this mean we're going to start using protobuf for our node requests?